tests/dofs/dof_tools_16a.cc

```diff
@@ -44,7 +45,8 @@ check_this(const DoFHandler<dim> &dof_handler)
   DoFTools::map_dof_to_boundary_indices(dof_handler, set, map);
 
   // create sparsity pattern
-  std::map<types::boundary_id, const Function<dim> *> boundary_ids;
+  std::map<types::boundary_id, const Function<dof_handler.dimension> *>
```

Comment:
I am having problems with this on older compilers. Can you please replace it here and in all the other places?
```suggestion
  std::map<types::boundary_id, const Function<DoFHandlerType::dimension> *>
```

Reply:
Fixed, I hope I greped all occurrences :)